src/memory.cpp

```diff
@@ -239,10 +242,12 @@ uint8_t debug_emu_read(uint8_t reg)
 		case 5: return gif_recorder_get_state();
 		case 6: return wav_recorder_get_state();
 		case 7: return Options.no_keybinds ? 1 : 0;
-		case 8: return (clockticks6502 >> 0) & 0xff;
-		case 9: return (clockticks6502 >> 8) & 0xff;
-		case 10: return (clockticks6502 >> 16) & 0xff;
-		case 11: return (clockticks6502 >> 24) & 0xff;
+		case 8: 
+		    clock_snap = clockticks6502 - clock_base;
```

Comment:
This line (latch) should probably not happen in the debug read routine.  The debug version of the read specifically exists so that it does not affect state.

Reply:
reverted this part